packages/traceability-schemas/scripts/schemas-to-vocab.js

```diff
@@ -76,15 +80,25 @@ const buildClass = (schema) => {
 
   const depList = dependencies.length
     ? `<b>Dependencies</b><ul>${dependencies
-        .map((key) => `<li><a href='#${key}'>${key}</a></li>`)
+        .map((key) => `<li>${key}</li>`)
```

Comment:
this was causing all the respec errors... snippets had no valid anchor definition.